app/scripts/shell/dimSearchFilter.directive.js

```diff
@@ -222,6 +223,27 @@
 
         return _.some(_duplicates.dupes, function(hash) { return item.hash === hash; });
       },
+      'unique': function() {
+          if (!_duplicates.hasOwnProperty('dupes')) {
+          var allItems = _.chain(dimStoreService.getStores())
```

Comment:
I think we might have a better way to get all items. Not 100% though. If we don't, I think we should.